app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -17,32 +17,82 @@ keywords:
   - teaching
   - teacher
   - special school
-quote:
-  q-jane-wilkinson:
-    text: "I specifically wanted to work with children identified with Special Educational Needs and Disabilities (SEND). I was fortunate to find a teacher training course provider that was able to allow me to practice in a mainstream school that also hosted a hearing impairment unit. And another school that offered support for children with a variety of learning needs. They were great at teaching me and letting me practice my classroom management skills early on. I was able to research and understand ways to differentiate resources for children who were on the autistic spectrum. And I was able to create and implement a scheme of work to reflect on and evaluate for later use once qualified."
-    name: "Jane Wilkinson, former teacher and teacher training adviser"
-    classes: quote--indent
+
 ---
-Most disabled pupils and pupils with special educational needs learn in mainstream schools.
+Special educational needs and disabilities (SEND) can affect pupils’ ability to learn. For example, they may struggle to read or write because they have dyslexia, or they may have difficulty concentrating. Some pupils may have physical impairments which means they need special equipment to help them learn.  
+
+Teaching pupils with SEND offers you a unique opportunity to make a lasting impact in their lives. You’ll help pupils overcome challenges and guide them to discover their strengths and fulfil their potential. Through your support, they’ll build self-esteem and resilience. 
+
+<blockquote class="quote quote--background-white quote--indent">
+  <p>I'm passionate about being there for my students, helping them see their potential and overcome their own challenges."</p>
+    <footer class="footer">
+      <div class="author">
+        <div>
+            <cite class="name">Gloria, lead maths practitioner</cite>
+        </div>
+      </div>
+</footer>
+</blockquote>
+
+Working with pupils with complex needs can be incredibly fulfilling. You’ll have the opportunity to: 
+
+* gain a deep understanding of diverse learning needs 
+* nurture a supportive learning environment  
+* advocate for inclusivity  
+
+Teachers who focus on supporting pupils with SEND are often called SEND teachers, but all teachers teach children with complex needs. Whether you train to teach in mainstream schools, special schools, or both, you’ll teach pupils with SEND, and learn how to support their needs.
+
+
+## SEND-focused teaching roles
+
+All initial teacher training courses cover teaching pupils with SEND. Once you have gained QTS, you can choose to specialise in teaching and nurturing pupils with SEND in a more SEND-focused role. This could be through:
```

Comment:
I think we need to say what QTS is? And maybe with a link to the QTS page? People may land on this page directly and may not know what we are talking about

```suggestion
All initial teacher training courses cover teaching pupils with SEND. Once you have gained qualified teacher status (QTS), you can choose to specialise in teaching and nurturing pupils with SEND in a more SEND-focused role. This could be through:
```

Reply:
If you have the QTS link here, you could remove it from the SENCO section?